linker-interface/shared/src/main/scala/org/scalajs/linker/interface/ESFeatures.scala

```diff
@@ -35,7 +35,8 @@ final class ESFeatures private (
     /* We define `val`s separately below so that we can attach Scaladoc to them
      * (putting Scaladoc comments on constructor param `val`s has no effect).
      */
-    _useECMAScript2015: Boolean,
+    _esVersion: ESVersion,
```

Comment:
The doc above states semantics about certain prefixes in here. We should probably revise the doc or stay consistent with the prefixes.